boards/mini_sam_m0/board_config.h

```diff
@@ -6,7 +6,7 @@
 #define VENDOR_NAME "Benjamin Shockley"
 #define PRODUCT_NAME "Mini SAM M0"
 #define VOLUME_LABEL "MINISAMBOOT"
-#define INDEX_URL "httpw://www.minifigboards.com"
+#define INDEX_URL "http://minifigboards.com"
```

Comment:
Did you mean `https:`? The other board is `https:`.

Reply:
Oh man...yes I did.

Reply:
I made the change. I'll merge it if it's all fine with you now.

Reply:
Yes, please and thank you!